### ISSUE TITLE
Bold Line Number doesn't work properly when caret is on Line 1

### ISSUE THREAD

**Issue by Eldaw** [microsoft/vscode#112353]
Version 1.52.0 re-introduced the bold line number (for the current line) that had been taken away a few versions ago.

But there is a small bug, where Line 1 doesn't get bolded correctly in 2 different scenarios:

Steps to reproduce:

1. Hit Ctrl+N to create a new empty file, which puts the keyboard caret on Line 1.
2. Notice that the '1' for the line number is not bolded, even though that is the current line. (this is the bug for Scenario 1)
3. Hit Enter. Notice that '2' gets bolded, as expected, because Line 2 is now the current line.
4. Hit the up arrow to return to line 1. Notice that now, the '1' gets bolded, as expected.
5. Use the mouse to select a different tab, and then go back to the tab that was active in the previous step, Step 4.
6. Notice that the '1' is no longer bolded. (this is the bug for Scenario 2)

In summary, when you start a new file, the '1' is not bolded. And when you have the caret on an empty Line 1, and switch to another tab and then come back, the '1' is not bolded.
